package/yast2-network.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu Feb 21 17:50:32 UTC 2019 - knut.anderssen@suse.com
+
+- When renaming a interface (bsc#1060207):
+  - Offer to the user the option to update device routes that
+    belongs to the previous interface name.
+  - Update the Routing devices list with the new one when needed.
+- Only delete a LanItem when canceling an addition (bsc#998540)
```

Comment:
this is more about internals, not that useful for user.

Reply:
Well, it fixes a bug

Reply:
this is usually my problem. Seems that you're also to deep already ;-)

e.g. something like "do not loose newly created virtual device when canceling edit action"